view/SSViewer.php

```diff
@@ -861,49 +861,47 @@ public static function set_theme($theme) {
 	/**
 	 * Traverses the given the given class context looking for templates with the relevant name.
```

Comment:
That description is a bit ambiguous now, we should clarify that it returns a list of potentially existing templates - it doesn't actually *look* for templates any more.